core/local/watcher.js

```diff
@@ -132,7 +133,7 @@ module.exports = class LocalWatcher {
     const started = new Promise((resolve) => {
       for (let eventType of ['add', 'addDir', 'change', 'unlink', 'unlinkDir']) {
         this.watcher.on(eventType, (path /*: ?string */, stats /*: ?fs.Stats */) => {
-          log.chokidar.debug({path}, eventType)
+          log.chokidar.debug({path, initialScan: !!this.initialScanEvents}, eventType)
```

Comment:
Wouldn't initializing `this.initialScan` before the `for` loop and using it for the condition do the trick?
I believe we [are][L183] [already][L222] [using][L258] [it][L280] to decide whether an initial scan is currently running.

[L183]: https://github.com/cozy-labs/cozy-desktop/blob/5a651744793946c56481339e7a2b59194f596957/core/local/watcher.js#L183
[L222]: https://github.com/cozy-labs/cozy-desktop/blob/5a651744793946c56481339e7a2b59194f596957/core/local/watcher.js#L222
[L258]: https://github.com/cozy-labs/cozy-desktop/blob/5a651744793946c56481339e7a2b59194f596957/core/local/watcher.js#L258
[L280]: https://github.com/cozy-labs/cozy-desktop/blob/5a651744793946c56481339e7a2b59194f596957/core/local/watcher.js#L280

Reply:
Got it: we want the `initialScan` property to be `false` as soon as `onFlush` is called so events occurring during analysis are not logged with `initialScan: true`.

Reply:
I'm still a bit uncomfortable with the addition of a new property to the already-bloated `LocalWatcher` class, though...